### DISCUSSION TITLE
SEB 3.3.3 on Mac / Permissions Required for Web Applications

### DISCUSSION THREAD

**Discussion by futellaa** [SafeExamBrowser/seb-mac#447] [Q&A]
Please help me :(

I'm trying to install seb on my Mac, but after it's installed, a notification like this appears.
![WhatsApp Image 2024-10-22 at 20 57 50](https://github.com/user-attachments/assets/08dff5c1-7c65-4c77-b3b5-ec94c36c0426)

"Permissions Required for Web Applications"
For this session, access for web applications is required. You need to authorize access before you can start this session.

After I click the OK button, the application automatically closes.
I’m confused about where to find the settings.

**Comment by danschlet** [comment]
There's a bug in SEB 3.3.3 regarding microphone and camera access for web applications. It should be fixed in [SEB 3.4,](https://github.com/SafeExamBrowser/seb-mac/releases) you can try it with the [release candidate version](https://github.com/SafeExamBrowser/seb-mac/releases/tag/3.4rc).
